src/adapters/streaming-key-ttl-checker.ts

```diff
@@ -0,0 +1,39 @@
+import { AppComponents } from '../types'
+import { IStreamingKeyChecker } from '../types/checker.type'
+import { CronJob } from 'cron'
+
+export async function createStreamingKeyTTLChecker(
+  components: Pick<AppComponents, 'logs' | 'sceneStreamAccessManager'>
+): Promise<IStreamingKeyChecker> {
+  const { logs, sceneStreamAccessManager } = components
+  const logger = logs.getLogger(`streaming-key-ttl-checker`)
+  let job: CronJob
+
+  async function start(): Promise<void> {
+    job = new CronJob(
+      '*/10 * * * *', // every 10 minutes
+      async function () {
+        try {
+          logger.info(`Looking into active streamings.`)
+
+          await sceneStreamAccessManager.removeExpiredStreamingKeys()
+        } catch (error) {
+          logger.error(`Error while checking places: ${error}`)
```

Comment:
I would recommend you to log the message error to prevent the error from polluting the logs:
```suggestion
          logger.error(`Error while removing expired streaming keys: ${isErrorWithMessage(error) ? error.message : 'Unknown error'}`)
```
I've also updated the message to talk about expired streaming keys.